src/lib/quiz/generate.ts

```
@@ -32,8 +39,9 @@ export function buildGenerationPrompt(
         "Generate a multiple choice question with exactly 4 options. Exactly one option must be correct.";
       break;
     case "true_false":
-      typeInstruction =
-        'Generate a true/false question. Make the statement false approximately half the time — when false, state something a student might plausibly believe but that is incorrect, and mark False as correct. Set options to [{text: "True", is_correct: ...}, {text: "False", is_correct: ...}].';
+      const targetLabel =
+        (trueFalseTarget ?? "false") === "true" ? "True" : "False";
```

Biome (2.4.11)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)